pom.xml

```diff
@@ -3,13 +3,13 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.exasol</groupId>
     <artifactId>kinesis-connector-extension</artifactId>
-    <version>1.1.3</version>
+    <version>1.1.4</version>
     <name>Exasol Kinesis Connector Extension</name>
     <description>Exasol Kafka Extension for accessing Apache Kafka</description>
     <url>https://github.com/exasol/kinesis-connector-extension/</url>
     <properties>
         <!-- Upgrade to 2.13.11 causes compile errors. This will be done in https://github.com/exasol/kinesis-connector-extension/issues/36 -->
-        <scala.version>2.13.3</scala.version>
+        <scala.version>2.13.11</scala.version>
```

Comment:
Is it possible to also upgrade to 2.13.12?

Reply:
Maybe, will check

Reply:
Upgraded